hpx/runtime/threads/policies/hwloc_topology_info.hpp

```diff
@@ -41,6 +42,35 @@ namespace hpx { namespace resource
 
 namespace hpx { namespace threads
 {
+    struct hpx_hwloc_bitmap_wrapper {
```

Comment:
This class will things make blow up if an instance of it is copied:
```
    hpx_hwloc_bitmap_wrapper hwlw1 (...);
    hpx_hwloc_bitmap_wrapper hwlw2 = hwlw1;
```
which will cause a double-free.

Please add proper copy constructor/assignment operator or alternatively proper move semantics.

Reply:
Alternatively you can make the class non-copyable (and non-movable).